core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -16,37 +16,7 @@
  */
 package org.apache.accumulo.core.client.impl;
 
-import java.io.IOException;
```

Comment:
What did you use for import manipulation?  I don't think the Accumulo project has declared how imports have been organized.  However I think we do try to organize imports using Eclipse's default settings inorder to cut down on noise generated by different IDEs organizing imports in different ways.  Can we just drop this portion of the patch?  Did the code add any imports?